platform/core/src/classes/MetadataProvider.ts

```diff
@@ -346,16 +334,13 @@ class MetadataProvider {
         break;
 
       case WADO_IMAGE_LOADER_TAGS.PATIENT_MODULE:
-        const { PatientName } = instance;
-
-        let patientName;
-        if (PatientName) {
-          patientName = formatPN(PatientName);
-        }
-
         metadata = {
-          patientName,
+          patientName: instance.PatientName,
```

Comment:
if we are changing the formatting of the PatientName (not doing formatPN) we should document why in migrations